src/Wallabag/CoreBundle/Resources/config/services.xml

```diff
@@ -21,6 +21,8 @@
                  class="Wallabag\CoreBundle\Security\Firewall\WsseListener" public="false">
```

Comment:
We need to choose which format we are going to use for declaring services, instead of mixing xml & yml. I'm more familiar with yml :)